CONTRIBUTING.rst

```diff
@@ -76,16 +76,25 @@ this, run::
 Local Testing
 -------------
 
-Please always run the style checks (see above) and the full test suite on
-your local computer before submitting a pull request, e.g.::
+Please always run the full test suite locally before submitting a pull
+request, e.g.::
 
-    $ git commit Bio/XXX.py Tests/test_XXX.py  -m "Fixed bug 123"
```

Comment:
according to the last sentence, we expect people to commit after the execution of the test suite